mirror-registry/README.md

```diff
@@ -1,106 +1,111 @@
+    ~~~
 
 ## **Development**
 
 In order to modify the registry, you will need:
 
-- git
-- make
-- podman
-- valid credentials for registry.redhat.io
+* git
+* make
+* podman
+* valid credentials for registry.redhat.io
 
-The repository for the code of the registry is stored at [https://github.com/quay/mirror-registry](https://github.com/quay/mirror-registry) , one can create its own modified image by
+The repository for the code of the registry is stored at [https://github.com/quay/mirror-registry](https://github.com/quay/mirror-registry), one can create its own modified image by:
 
-- authenticating to registry.redhat.io from a valid pull secret
+1. Authenticating to registry.redhat.io from a valid pull secret
 
-```
-PULL_SECRET="/root/openshift_pull.json"
-REDHAT_CREDS=$(cat $OPENSHIFT_PULL_JSON | jq .auths.\"registry.redhat.io\".auth -r | base64 -d)
-RHN_USER=$(echo $REDHAT_CREDS | cut -d: -f1)
-RHN_PASSWORD=$(echo $REDHAT_CREDS | cut -d: -f2)
-podman login -u "$RHN_USER" -p "$RHN_PASSWORD" registry.redhat.io
-```
+    ~~~sh
+    PULL_SECRET="/root/openshift_pull.json"
+    REDHAT_CREDS=$(cat $OPENSHIFT_PULL_JSON | jq .auths.\"registry.redhat.io\".auth -r | base64 -d)
+    RHN_USER=$(echo $REDHAT_CREDS | cut -d: -f1)
+    RHN_PASSWORD=$(echo $REDHAT_CREDS | cut -d: -f2)
+    podman login -u "$RHN_USER" -p "$RHN_PASSWORD" registry.redhat.io
+    ~~~
 
-- creating a tar with all the contents( the dependency images get bundled)
+2. creating a tar with all the contents(the dependency images get bundled):
 
-```
-make build-online-zip
-```
+    ~~~sh
+    make build-online-zip
+    ~~~
 
 ## **Limitations/Additional Notes**
 
-- the command can also target a remote host, although we didnt see too much benefit to going with such approach.
-- It was not possible to specify a different user than init in the first release. We opened [https://github.com/quay/mirror-registry/pull/46](https://github.com/quay/mirror-registry/pull/46) to address it, which got merged so this will be possible in an upcoming release.
-- IPV6 isn't currently supported by quay. We opened [https://github.com/quay/mirror-registry/pull/47](https://github.com/quay/mirror-registry/pull/47) to address it by using a side haproxy container, but this got closed. For the time beeing, the folder [ipv6](ipv6) contains assets allowing to deploy an extra container in an ipv6 context. This can be deployed with the following instructions:
+* The command can also target a remote host, although we didn't see too much benefit to going with such approach.
+* IPv6 isn't currently supported by quay. We opened [https://github.com/quay/mirror-registry/pull/47](https://github.com/quay/mirror-registry/pull/47) to address it by using a side HAProxy container, but this got closed. For the time being, the folder [IPv6](ipv6) contains assets allowing to deploy an extra container in an IPv6 context. This can be deployed with the following instructions:
```

Comment:
since the PR got closed we can avoid referencing it

Reply:
Done.

Reply:
i meant both PRS